src/component/huskelapp/redigering/huskelapp-slett-arbeidsliste.tsx

```diff
@@ -0,0 +1,83 @@
+import { Alert, BodyShort, Button, Heading } from '@navikt/ds-react';
+import { TrashIcon } from '@navikt/aksel-icons';
+import { useState } from 'react';
+import { useAppStore } from '../../../store/app-store';
+import { useDataStore } from '../../../store/data-store';
+import { logMetrikk } from '../../../util/logger';
+import { trackAmplitude } from '../../../amplitude/amplitude';
+import { slettArbeidsliste } from '../../../api/veilarbportefolje';
+import { ifResponseHasData } from '../../../util/utils';
+
+export const SlettArbeidsliste = () => {
+    const [visSlettebekreftelse, setVisSlettebekreftelse] = useState(false);
+    const [loading, setLoading] = useState(false);
+    const [error, setError] = useState(false);
+
+    const { brukerFnr } = useAppStore();
+    const { setArbeidsliste } = useDataStore();
+
+    const handleSlettArbeidsListe = () => {
+        logMetrikk('visittkort.metrikker.fjern_arbeidsliste');
+        trackAmplitude({
+            name: 'knapp klikket',
+            data: { knapptekst: 'Fjern arbeidsliste', effekt: 'Fjern bruker fra arbeidslista' }
+        });
+        setLoading(true);
+
+        slettArbeidsliste(brukerFnr)
+            .then(ifResponseHasData(setArbeidsliste))
+            .then(() => setLoading(false))
+            .catch(() => setError(true))
+            .then(() => setVisSlettebekreftelse(false));
+    };
+
+    return (
+        <>
+            {error && (
+                <Alert variant="error" size="small" className="sletting-av-arbeidsliste-feilet">
+                    Noe gikk galt ved sletting av arbeidslista.
+                </Alert>
+            )}
+            {!visSlettebekreftelse && (
+                <Button
+                    onClick={() => {
+                        setVisSlettebekreftelse(true);
+                        setError(false);
+                    }}
+                    size="xsmall"
+                    variant="tertiary"
+                    icon={<TrashIcon aria-hidden />}
+                >
+                    Slett
+                </Button>
+            )}
+            {visSlettebekreftelse && (
+                <div className="slettArbeidslisteContainer">
+                    <Heading size="xsmall" as="h3">
```

Comment:
Vet ikke om det gjør noen forskjell, men det er innebygget i [Heading-komponenten](https://aksel.nav.no/komponenter/core/typography#headingdemo-level) med `level`!

```suggestion
                    <Heading size="xsmall" level="3">
```